views/postgres.ejs

```diff
@@ -0,0 +1,194 @@
+<%- include('./partials/head') %>
+<h1 id="0-credentials">0 Credentials</h1>
+<pre><code><span class="hljs-symbol">hostname:</span> <span class="hljs-number">165.227</span><span class="hljs-number">.55</span><span class="hljs-number">.105</span>
+<span class="hljs-symbol">username:</span> <%= username %>
+<span class="hljs-symbol">password:</span> <%= dbPassword %>
+</code></pre><p>or </p>
+<p>Create PG Credentials - Button</p>
+<h1 id="1-intro">1 Intro</h1>
+<p>Postgres is an application. More specifically, a database application for you to organize your data. It can store data for you and you can have multiple apps connect to your Postgres:</p>
+<div style="background-color: white; text-align: center"><img src="https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggTFJcblx0RFtBcHAxXSAtLT4gUG9zdGdyZXNcblx0RVtBcHAyXS0tPiBQb3N0Z3JlcyBcblx0RltBcHAzXSAtLT4gUG9zdGdyZXMiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ" alt="multiple apps connecting to postgres"></div>
+<h1 id="2-basics">2 Basics</h1>
+<p>To quickly help you get started, we will go over a few basic commands that you can follow along. Explanations and best practices will be covered in the section below.</p>
+<h2 id="2-1-connect">2.1 Connect</h2>
+<p>To connect to your app to Postgres, you need a module called <code>pg</code>. You need to specify where the postgres application is located (host and port), the name of your database, and your username / password to login to the database.</p>
+<pre><code><span class="hljs-keyword">const</span> { Client } = <span class="hljs-built_in">require</span>(<span class="hljs-string">'pg'</span>)
+
+<span class="hljs-keyword">const</span> startApp = <span class="hljs-keyword">async</span> ()=&gt;{
+    client = <span class="hljs-keyword">new</span> Client({
+            <span class="hljs-attr">host</span>: <span class="hljs-string">'&lt;host name&gt;'</span>,
+                <span class="hljs-attr">port</span>: <span class="hljs-number">5432</span>,
+                    <span class="hljs-attr">user</span>: <span class="hljs-string">'&lt;username&gt;'</span>,
+                        <span class="hljs-attr">password</span>: <span class="hljs-string">'&lt;password&gt;'</span>,
+                            <span class="hljs-attr">database</span>: <span class="hljs-string">'&lt;database_name&gt;'</span>
+                              })
```

Comment:
Is there any way you can format this more so it's easier to read?

Reply:
I'm considering to use markedjs. I'll update soon